packages/ui-extensions/src/surfaces/admin/components/Box/examples/responsive-shipping-notice.html

```diff
@@ -0,0 +1,8 @@
+<s-box
+  padding="large-200"
```

Comment:
```suggestion
  padding="@container (inline-size < 500px) small, large"
```
Should this be using the [responsive syntax](https://shopify-dev.shop.dev/docs/api/app-home/using-polaris-components#responsive-values)?